src/Core/AdminConsole/Services/Implementations/PolicyService.cs

```diff
@@ -256,6 +264,15 @@ private async Task HandleDependentPoliciesAsync(Policy policy, Organization org)
         }
     }
 
+    private async Task HasVerifiedDomainsAsync(Organization org)
```

Comment:
Please add a unit test to check this on https://github.com/bitwarden/server/blob/main/test/Core.Test/AdminConsole/Services/PolicyServiceTests.cs